build-tools/packages/build-tools/src/typeValidator/compatibility.ts

```diff
@@ -115,9 +115,11 @@ namespace Test2 {
 }
 
 {
-	// This really seems like it shouldn't compile, but maybe its ok for non const enums since the actual values are
-	// Enums apparently just check that the names (of both the enum and its members) and the values of its members match, but not that the pairing of them is the same.
+	// @ts-expect-error Changing the value of an enum member breaks assignability (this changed somewhere between TypeScript 5.1 and 5.4).
 	type _check = requireAssignableTo<Test1.A, Test2.A>;
+	// TypeOnly does not consider renumbering the Enum to be a breaking change.
+	// Maybe this is ok for non const enums since the references should be to the enum member not the constant.
+	type _check4 = requireAssignableTo<TypeOnly<Test1.A>, TypeOnly<Test2.A>>;
```

Comment:
I don't understand the numbering scheme for the variables here, and this is not exactly inconsistent with the rest, but just want to double-check this is the variable name you intended

Reply:
I need an unused identifier so the compiler wouldn't complain what it is does not matter and 2 and 3 were already taken.